app/views/articles/_actions.html.erb

```diff
@@ -36,7 +36,7 @@
         <clipboard-copy for="article-copy-link-input" aria-live="polite" aria-controls="article-copy-link-announcer" class="dropdown-link-row">
           <div class="flex items-center">
             <input type="text" id="article-copy-link-input" value="<%= article_url(@article) %>" class="crayons-textfield" readonly />
-            <%= inline_svg_tag("copy.svg", aria: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: "Notifications") %>
+            <%= inline_svg_tag("copy.svg", aria: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: "Copy this link to the clipboard") %>
```

Comment:
suggestion (not-blocking): This feature isn't currently keyboard accessible (I'll make a separate issue for that), but it might be helpful, especially for low vision users, to be explicit that the displayed link is the article's URL.

```suggestion
            <%= inline_svg_tag("copy.svg", aria: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: "Copy article link to the clipboard") %>
```

Reply:
I hesited to write like you with 'article' for more accessibility and not to be confused when you read that

Reply:
Thomas, not a problem. Since I just merged this PR a moment ago, before I saw the suggestion here, I'm opening that small suggested change in a separate PR now.

Reply:
Good I see that!